ethereum/circuits/lib/src/verifiers/account_test.nr

```diff
@@ -79,20 +79,20 @@ fn test_assert_account_equals_code_hash_with_wrong_value() {
 
 #[test]
 fn test_verify_account() {
-    let _ = verify_account(state_proof.key, account, state_proof, state_root);
+    let _ = verify_account(address, account, state_proof_input, state_root);
 }
 
 #[test(should_fail_with="Address mismatch")]
 fn test_verify_account_wrong_address() {
-    let mut wrong_address = state_proof.key;
+    let mut wrong_address = address;
     wrong_address[0] += 1;
-    let _ = verify_account(wrong_address, account, state_proof, state_root);
+    let _ = verify_account(wrong_address, account, state_proof_input, state_root);
 }
 
 // TODO: Add correct assertion message when noir-trie-proofs is updated
```

Comment:
Is this still a TODO? Seems like we don't even use noir-trie-proofs any more

Reply:
You're right, added correct assertion message